kie-internal/src/main/java/org/kie/internal/xstream/XStreamUtils.java

```diff
@@ -32,7 +32,8 @@
 public class XStreamUtils {
 
     private static final String[] WHITELISTED_PACKAGES = new String[] {
-        "org.drools.core.command.**"
+        "org.drools.core.command.**",
+        "org.drools.core.runtime.impl.ExecutionResultImpl"
```

Comment:
@mswiderski now I'm seeing there are a few other tests failing caused by a missing permission for "org.drools.core.common.DefaultFactHandle". I think that one and also EventFactHandle should even be added here.